core/tests/protractor_utils/CollectionEditorPage.js

```diff
@@ -76,9 +76,8 @@ var CollectionEditorPage = function() {
       by.cssContainingText('.dropdown-menu', new RegExp(query)));
     await waitFor.presenceOf(
       dropdownResultElement, 'Unable to find exploration: ' + query);
-
-    var matchingSearchResult = element(by.cssContainingText(
-      '.uib-typeahead-match', new RegExp(query)));
+    var matchingSearchResult = (
+      element(by.cssContainingText('.ngb-highlight', query)));
```

Comment:
Can you make this a chained selector rooted at a `protractor-test-*` element? This requirement is [described in the wiki](https://github.com/oppia/oppia/wiki/End-to-End-Tests#selecting-elements)

Reply:
Can't be done due to reasons specified here. https://github.com/oppia/oppia/pull/12563#discussion_r697971790